h/stats.py

```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
-import statsd
-
-__all__ = ("get_client",)
-
-
-def get_client(settings):
-    return statsd.StatsClient(
-        host=settings.get("statsd.host", "localhost"),
-        port=settings.get("statsd.port", 8125),
-        prefix=settings.get("statsd.prefix", ""),
-    )
-
-
-def includeme(config):
-    # Allow easy access to a statsd client as `request.stats`
-    config.add_request_method(
-        lambda r: get_client(r.registry.settings), name="stats", reify=True
-    )
```

Comment:
`stats.py` adds the `request.stats` object (a statsd client) that code all over the place uses to report stats. Apparently we used to use Python's logging module for metrics. This `stats.py` file / `request.stats` object was [added when we first moved from `logging` to `statsd`](https://github.com/hypothesis/h/pull/1616/)